src/platform/web/ui/login/PasswordLoginView.js

```diff
@@ -18,7 +18,7 @@ import {TemplateView} from "../general/TemplateView";
 
 export class PasswordLoginView extends TemplateView {
     render(t, vm) {
-        const disabled = vm => !!vm.isBusy;
+        const disabled = vm => vm.isBusy && vm.isEnabled;
```

Comment:
This seems weird. We disable the form when we're both busy and the view model says we should be enabled rather than disabled? Do we even need the two fields?

Reply:
Yes, it should be `vm.isBusy || !vm.isEnabled;` sorry ^^'